### PR TITLE
[LDR R1] EOS-13734 - S3 Audit log fix

### PR DESCRIPTION
**Pull request by shanbhagsv1**
For the initial review.

### CODE REVIEW

File: server/s3_get_bucket_action_v2.cc [hunk 1]
```diff
@@ -145,8 +146,7 @@ void S3GetBucketActionV2::send_response_to_s3_client() {
                   request->get_bucket_name());
     std::string& response_xml = error.to_xml();
     request->set_out_header_value("Content-Type", "application/xml");
-    request->set_out_header_value("Content-Length",
```
Comment (shanbhagsv1) [inline server/s3_get_bucket_action_v2.cc:148]: Fix this

**Review by amitwac2608 — CHANGES_REQUESTED** [review]
you need to add UT and/or ST to validate that the for GET object request, the bytes_sent has value same as PUT object request's object size/content-length

**Comment by shanbhagsv1** [comment]
@amitwac2608  Audit log is currently disabled on DEV will enable it if needed created new story https://jts.seagate.com/browse/EOS-13756. Please add the required UT/ST's

**Comment by shanbhagsv1** [comment]
Build job http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/S3server/32/

**Comment by shanbhagsv1** [comment]
Retrigged build due to new fixes http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/S3server/33/

**Review by knrajnambiar76 — APPROVED** [review]

File: server/s3_get_bucket_policy_action.cc [hunk 2]
```diff
@@ -93,8 +93,9 @@ void S3GetBucketPolicyAction::send_response_to_s3_client() {
     request->send_response(error.get_http_status_code(), response_xml);
 
   } else {
-    request->send_response(S3HttpSuccess200,
-                           bucket_metadata->get_policy_as_json());
+    std::string& response_xml = bucket_metadata->get_policy_as_json();
```
Comment (amitwac2608) [inline server/s3_get_bucket_policy_action.cc:96]: its better to take the return value in const reference

**Review by amitwac2608 — CHANGES_REQUESTED** [review]

**Review by amitwac2608 — COMMENTED** [review]
rest looks good. Just remember to ask QA to check if they have the test case in their suite of tests or not. I f not ask them to add the test case.

File: server/s3_get_bucket_policy_action.cc (hunk 2 above)
Comment (shanbhagsv1) [reply server/s3_get_bucket_policy_action.cc:96]: Lets fix it as part of https://jts.seagate.com/browse/EOS-13756 now. Need retriggering of jenkins again for this fix.

**Review by nileshgovande — APPROVED** [review]

File: server/s3_get_bucket_policy_action.cc (hunk 2 above)
Comment (amitwac2608) [reply server/s3_get_bucket_policy_action.cc:96]: sure. I assume you have already updated the ticket so that we do not miss this.

**Pull request merged**